script/download_map.sh

```diff
@@ -9,7 +9,7 @@ python3 -m pip install --upgrade gdown
 if [ ! -d $MAP_PATH ]; then
     echo "Download map from Internet..."
     mkdir $MAP_PATH
-    gdown --fuzzy -O $MAP_PATH/lanelet2_map.osm https://drive.google.com/file/d/1vm9SvalJe7Bc9sh_8jK-0cPulVAov5uD/view
+    gdown --fuzzy -O $MAP_PATH/lanelet2_map.osm https://drive.google.com/file/d/1TBfWKDxxGnfm1ZUfzotDgcadEERYr85s/view
```

Comment:
I preferred to download the map to another folder and change the map path by the following configuration
https://github.com/evshary/autoware_carla_launch/tree/main/carla_map#change-the-map